src/Roslyn.Diagnostics.Analyzers/UnitTests/CSharpAvoidOptSuffixForNullableEnableCodeTests.cs

```diff
@@ -264,5 +265,99 @@ public void Method1({|CS0246:Class2|}? s)
 }",
             }.RunAsync();
         }
+
+        [Fact, WorkItem(3813, "https://github.com/dotnet/roslyn-analyzers/issues/3813")]
+        public async Task RS0046_CSharp8_NullableEnabledCode_InterfaceImplementation_NoDiagnostic()
+        {
+            await new VerifyCS.Test
+            {
+                LanguageVersion = LanguageVersion.CSharp8,
+                TestCode = @"
+#nullable enable
+
+public interface ISomething
+{
+    void Method1(string? [|sOpt|]);
+}
+
+public class Class1 : ISomething
+{
+    public void Method1(string? sOpt)
+    {
+        string? [|localOpt|] = null, [|otherLocalOpt|] = null;
+
+        System.Console.WriteLine(""{0}, {1}, {2}"", sOpt, localOpt, otherLocalOpt);
+    }
+}",
+                FixedCode = @"
+#nullable enable
+
+public interface ISomething
+{
+    void Method1(string? s);
+}
+
+public class Class1 : ISomething
+{
+    public void Method1(string? sOpt)
```

Comment:
Is it possible to force the renaming of implementation/overrides?

Reply:
Oh sorry, that seems to be only for the overloads. I don't think functionality exists for implementation/overrides.